resources/session02/homework/http_server.py

```diff
@@ -32,10 +35,25 @@ def parse_request(request):
         raise NotImplementedError("We only accept GET")
     return uri
 
-
 def resolve_uri(uri):
     """This method should return appropriate content and a mime type"""
-    return b"still broken", b"text/plain"
+    homedir = str(pathlib.Path.cwd().joinpath('webroot'))
+    fullpath = pathlib.Path(homedir + uri)
+    print('\nFull path: {}'.format(fullpath))
+    if fullpath.exists():
+        mimetype = mimetypes.guess_type(uri)[0]
```

Comment:
This might be a bit premature for looking up the mimetype this way.  If uri is a directory path, then the tuple you get back is `(None, None)`, which doesn't do you any good.  You might consider not doing this until you are certain what you have is a file.